MAUI/Calendar/getting-started.md

```diff
@@ -130,11 +130,11 @@ this.Calendar.MonthView.FirstDayOfWeek = DayOfWeek.Monday;
 
 ## Date selection
 
-The Calendar allows the user to select a single date, multiple dates, or a range of dates by interaction or programmatic selection. The default selection mode is `Single`.
+Calendar allows the user to Select a single date, multiple dates, or a range of dates through interaction and programmatically. The default selection mode is `Default`. 
```

Comment:
The Calendar allows the user to select a single date, multiple dates, or a range of dates by interaction or programmatic selection. The default selection mode is `Default.`

Reply:
Content changed for date selection